package.json

```diff
@@ -30,13 +30,9 @@
     "prettier-plugin-ember-template-tag": "^1.1.0",
     "release-plan": "^0.9.0"
   },
+  "packageManager": "pnpm@9.1.1",
```

Comment:
The `pnpm/action-setup` [supports](https://github.com/pnpm/action-setup?tab=readme-ov-file#version) this field already, hence replacing `with version` blocks with single declaration.